code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -75,14 +83,28 @@
 		H.update_inv_l_hand(0)
 		H.update_inv_r_hand()
 
-	playsound(src.loc, 'sound/weapons/gun_empty.ogg', 15, 1)
+	playsound(user, 'sound/weapons/gun_empty.ogg', 25, sound_range = 5)
 	add_fingerprint(user)
 
 	if(blood_overlay && blood_color)
 		overlays.Cut()
 		add_blood(blood_color)
 	return
 
+/obj/item/weapon/melee/telebaton/proc/stun(mob/living/carbon/human/target, mob/living/user)
+	if(target.check_shields(src, 0, "[user]'s [name]"))
+		return FALSE
+	user.visible_message(SPAN_DANGER("[user] knocks down [target] with \the [src]!"),\
```

Comment:
This always says "knocks down" even if you don't actually knock down target. Either need to be made more universal or do a check post-hit whether target is stunned/lying and show respective message.